src/server/index.jsx

```diff
@@ -73,32 +99,17 @@ server
       }
     });
   })
-  .get('/*', async ({ url }, res) => {
+  .get(articleRegexPath, async ({ url }, res) => {
```

Comment:
Im not convinced by this, unless its only short term untill we properly handle unmatched route error handling

Reply:
@dr3 could you open an issue to re-evaluate this in the future, can be assessed after we launch v1.0